docs/learn/learn-async-backing.md

```diff
@@ -7,100 +7,111 @@ keywords: [parachains, slots, backing, parablock]
-rushing to be backed and included within this two-block window due to the inherent requirement for
-synchrony between the parachain and relay chain.
-
-## Asynchronous Backing on Polkadot
-
-With asynchronous backing, the window of time is customizable and will most likely sit around the
-6-18 second range. It also introduces a parameter to aid in defining the maximum amount of ancestor
-blocks, which allows for a parablock to be backed later in the future, enabling more computational
-and storage time per block. The context for the latest parablock is derived from the unincluded
-segment of block ancestors upon which the newest parablock is built.
-
-These blocks can be prepared in anticipation of being included later rather than keeping in sync
-with the relay chain's progress 1-1. The block can be in a different stage from another block being
-built, as long as it abides by the parameters set forth by the asynchronous backing configuration.
+Asynchronous backing brings the following changes to the parachain protocol:
+
+- parablock generation is decoupled from the backing process, this avoids rushing through the
+  backing process in six seconds and gives more time to collators to properly use blockspace,
+  decreasing the probability of generating empty parablocks
+- parachains can generate blocks and have them cashed (or stacked) into **unincluded segments** of
+  parablock ancestors (i.e. parablocks that are seconded but not yet backed by paravalidators and
+  included in the relay chain) rather than ancestors included in the relay chain state.
+
+These changes allow to generate parablock `P + 1` soon after parablock `P` has been generated and
+cashed into the unicluded segment within the first six-second time window. At the same time,
+parablock ancestor `P - 1`, pushed to the unincluded segment before `P`, is backed and included in
+the relay chain.
+
+FIGURE HERE?
+
+The **contextual execution** (which is the context built by block ancestors used by the parachains
+to generate new parablocks) shifts from being the latest included parent block in the relay chain,
+to being the latest ancestor parablock pushed into the unincluded segment. This allows to start
+building parablocks earlier, giving plenty of time to parachain collators to fit more transactions
+and prepare block candidates for backing and inclusion.
+
+The execution context for the latest parablock is thus derived from the unincluded segment of block
+ancestors upon which the newest parablock is built. These blocks can be prepared in anticipation of
+being included later rather than keeping in sync with the relay chain's progress 1-1. A parablock
+can be in a different stage from another one being built if it abides by the parameters set forth by
+the asynchronous backing configuration.
+
+Asynchronous backing also introduces a parameter to define the maximum number of ancestor blocks
```

Comment:
As part of the reorg, should we maybe categorize this as "Further Benefits" or similar?